ciongke/cmd/hook/main.go

```diff
@@ -73,6 +73,22 @@ type Event struct {
 	Payload []byte
 }
 
+// kubeClient is satisfied by kube.Client.
+type kubeClient interface {
+	ListPods(labels map[string]string) ([]kube.Pod, error)
+	DeletePod(name string) error
+
+	ListJobs(labels map[string]string) ([]kube.Job, error)
+	CreateJob(j kube.Job) (kube.Job, error)
+	DeleteJob(name string) error
+}
+
+// githubClient is satisfied by github.Client.
+type githubClient interface {
```

Comment:
Should I move the interface definitions closer to the functions that use them? Right now I have them by the struct that includes them.